minecraft-server/start-minecraft.sh

```diff
@@ -178,31 +211,135 @@ esac
 fi
 
 if [ ! -e server.properties ]; then
+  echo "Creating server.properties"
   cp /tmp/server.properties .
 
   if [ -n "$WHITELIST" ]; then
+    echo "Creating whitelist"
     sed -i "/whitelist\s*=/ c whitelist=true" /data/server.properties
     sed -i "/white-list\s*=/ c white-list=true" /data/server.properties
   fi
 
   if [ -n "$MOTD" ]; then
```

Comment:
It looks like we need a helper function now for all the repeated sed blocks. I can refactor that later if you didn't want to do that.